docs/config-file-resolutions-yml.md

```diff
@@ -0,0 +1,94 @@
+# The `resolutions.yml` file
+
+Resolutions allow *resolving* any errors or policy violations by marking them as acceptable and can be ignored.
+
+You can re-use [examples/resolutions.yml](examples/resolutions.yml) as the base file for your scans.
+
+### When to Use Resolutions
+Resolutions should be used when it impossible to solve an issue or is planned to be fixed at a later time.
```

Comment:
or is planned to be fixed at a later time -> or a fix is planned for a later time

Reply:
"it **is** impossible", "**it** is planned"

Reply:
done